access-control-app/src/cmr/access_control/services/acl_service.clj

```diff
@@ -357,3 +360,39 @@
     (if (or (nil? user-token)(re-matches #"^[\s\t]*$" user-token))
       (auth-util/get-sids context :guest)
       (auth-util/get-sids context (tokens/get-user-id context user-token)))))
+
+(defn- fetch-s3-buckets-by-groups
+  [context groups providers]
+  (let [base-query (gc/or-conds (map #(qm/string-condition :permitted-group-ids
```

Comment:
why do we match group-id with wildcards?

Reply:
This was copied from another query that did something similar, it's probably not necessary.